Cartfile

```diff
@@ -1,5 +1,5 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" "16.0.0"
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 60.0
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" ~> 16.2.0
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 62.0
 github "mapbox/mapbox-directions-swift" "v2.0.0-beta.7"
```

Comment:
During Mapbox Navigation release process Mapbox Directions will have to be released to depend on Turf `v2.0.0-rc.1`.

/cc @azarovalex

Reply:
Thanks for taking care of the dependencies, I'll take a look at the Turf release